README.md

```diff
@@ -128,18 +128,22 @@ make test
 
 Compiling CX on Windows requires a recent version of Git to be installed. 
 
-Before compiling CX, install dependencies with `pacman` (download MSYS2):
+Pacman is a utility which manages software packages.   
 
-```
-pacman -Sy
+To install pacman, download [Mysys2](https://www.msys2.org) and run the installer.
+
+When the installation is complete, click ` Run MSYS2 now `.     
+
+If MSYS2 has already been installed, run it through the start menu.
 
-pacman -S mingw-w64-x86_64-openal
+You can run a full system upgrade and install required dependencies with: 
+
+```
+pacman -Syu git mingw-w64-x86_64-openal base-devel mingw-w64-x86_64-toolchain
```

Comment:
I have tested this command, works fine. 
It installs all of required dependencies.